cypress/support/step-definitions/multi-action-button-steps.js

```diff
@@ -23,20 +22,34 @@ Then('Multi Action Button state is not disabled', () => {
 });
 
 Then('Multi Action Button as on preview is {string}', (asProperty) => {
-  multiActionButtonPreview()
-    .children()
-    .should('have.class', `${MULTI_ACTION_BUTTON_AS_PROPERTY}${asProperty}`);
+  switch (asProperty) {
+    case 'primary':
+      multiActionButtonPreview().children()
+        .should('have.css', 'background-color', 'rgb(30, 73, 159)');
```

Comment:
is it possible to add other css condition?

Reply:
fixed